.env

```diff
@@ -0,0 +1,2 @@
+DATABASE_URL_DEV= postgres://ktortwjp:ZST0lAva67i-uqCgSHc88ufJZ6sKl8-5@stampy.db.elephantsql.com:5432/ktortwjp
```

Comment:
Yo! This is the .env file which is meant to be gitignored. . . I don't expect to see it here

Reply:
My mistake, ignoring it right now, thank you.